lib/Swagger/src/ContentEntity/Models/Author.php

```diff
@@ -0,0 +1,339 @@
+{
+    /**
+      * The original name of the model.
+      * @var string
+      */
+    protected static $swaggerModelName = 'Author';
+
+    /**
+      * Array of property to type mappings. Used for (de)serialization
+      * @var string[]
+      */
+    protected static $swaggerTypes = array(
+        'id' => 'string',
+        'username' => 'string',
+        'user_type' => 'string',
+        'external_user_id' => 'string',
+        'avatar' => 'map[string,string]'
+    );
+
+    public static function swaggerTypes()
+    {
+        return self::$swaggerTypes;
+    }
+
+    /**
+     * Array of attributes where the key is the local name, and the value is the original name
+     * @var string[]
+     */
+    protected static $attributeMap = array(
+        'id' => 'id',
+        'username' => 'username',
+        'user_type' => 'userType',
+        'external_user_id' => 'externalUserId',
+        'avatar' => 'avatar'
+    );
+
+    public static function attributeMap()
+    {
+        return self::$attributeMap;
+    }
+
+    /**
+     * Array of attributes to setter functions (for deserialization of responses)
+     * @var string[]
+     */
+    protected static $setters = array(
+        'id' => 'setId',
+        'username' => 'setUsername',
+        'user_type' => 'setUserType',
+        'external_user_id' => 'setExternalUserId',
+        'avatar' => 'setAvatar'
+    );
+
+    public static function setters()
+    {
+        return self::$setters;
+    }
+
+    /**
+     * Array of attributes to getter functions (for serialization of requests)
+     * @var string[]
+     */
+    protected static $getters = array(
+        'id' => 'getId',
+        'username' => 'getUsername',
+        'user_type' => 'getUserType',
+        'external_user_id' => 'getExternalUserId',
+        'avatar' => 'getAvatar'
+    );
+
+    public static function getters()
+    {
+        return self::$getters;
+    }
+
+    
+
+    
+
+    /**
+     * Associative array for storing property values
+     * @var mixed[]
+     */
+    protected $container = array();
+
+    /**
+     * Constructor
+     * @param mixed[] $data Associated array of property value initalizing the model
+     */
+    public function __construct(array $data = null)
+    {
+        $this->container['id'] = isset($data['id']) ? $data['id'] : null;
+        $this->container['username'] = isset($data['username']) ? $data['username'] : null;
+        $this->container['user_type'] = isset($data['user_type']) ? $data['user_type'] : null;
+        $this->container['external_user_id'] = isset($data['external_user_id']) ? $data['external_user_id'] : null;
+        $this->container['avatar'] = isset($data['avatar']) ? $data['avatar'] : null;
+    }
+
+    /**
+     * show all the invalid properties with reasons.
+     *
+     * @return array invalid properties with reasons
+     */
+    public function listInvalidProperties()
+    {
+        $invalid_properties = array();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Immediately return this expression instead of assigning it to the temporary variable "$invalid_properties". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1488)